app/common/update.py

```diff
@@ -253,3 +258,65 @@ def merge_local_db():
 
         logger.info(str(records_inserted) + " records were inserted into local db.")
         logger.info(str(records_updated) + " records in local db were updated.")
+        
+
+def download_dat_files():
+    config = load_user_config()
+    valid_directory = False
+    
+    if not config["config"]["installdirectory"]:
+    
+        #First, let's check the default location and see if it's installed there
+        path = 'C:/Program Files (x86)/SquareEnix/DRAGON QUEST X/Game/Content/Data'
+        
+        if os.path.exists(path):
+            update_user_config('config', 'installdirectory', path)
+        else:
+            #Doesn't exist. Let's prompt the user for it.
+            warning_message(
+            title="[dqxclarity] Couldn't Find Directory",
+            message="Could not find DQX directory. Please select\n\nthe Data folder located in your\n\nDQX install location."
+        )
+            
+            path = askdirectory()

```

Comment:
We should use a different variable name than `path`. This means something else in linux (I am not sure if Windows shares the same). `dqx_path` or whatever you feel is best here.